packages/web-twig/src/Resources/components/Accordion/Accordion.twig

```diff
@@ -0,0 +1,22 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _elementType = (props.elementType is defined) ? props.elementType : 'section' -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Accordion' -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id=' ~ _id ~ '' : null -%}
```

Comment:
ok, fixed

Reply:
I do not see the double quotes? And currently isn't the last `''` useless?

Reply:
@adamkudrna @crishpeen @literat  Since half of the components have quotes in the API and the rest don't. And that I've already changed and reverted it once, so I'm going to leave it in this state and then all the other occurrences would go through and it would be unified, do you agree?

Reply:
We should have them everywhere. As mentioned below, you can see how it misbehaved in the snapshot. I prefer to add them here now and create a new issue to add them everywhere.

Reply:
Ok, I'll add them again to all of them and then go through all the others to fix it.